string level
string shy_away_position